data/timeline.json.unvalidated.in

```diff
@@ -3158,7 +3158,7 @@
             "type": "chat-actor",
             "data": {
                 "actor": "Ada",
-                "message": "Now that you've learned a bit about what runs your system and the files in your system, how about we start learning a programming language so that you can write your own programs? I've added a new lesson to showmehow to start exploring the Python programming language. Check it out."
+                "message": "Now that you’ve learned a bit about your system, let’s learn about Python, a programming language that lets you write your own programs. To get started, open the Terminal app on your desktop and type 'showmehow python'"
```

Comment:
Oops, must have been a copy-paste thing. I thought I had gotten rid of these, but maybe not :)